HuntHelper/Gui/HuntTrainUI.cs

```diff
@@ -406,6 +407,11 @@ private void DrawImportWindowModal()
         ImGui.SetNextWindowPos(center, ImGuiCond.Appearing, new Vector2(0.5f, 0.5f));
         ImGui.SetNextWindowSize(new Vector2(400, 700), ImGuiCond.FirstUseEver);
 
+        if (_openImportPopup)
+        {
+            ImGui.OpenPopup($"{GuiResources.HuntTrainGuiText["ImportWindowTitle"]}##popup");
+            _openImportPopup = false;
+        }
```

Comment:
i don't love this, but just calling `ImGui.OpenPopup()` from `OpenImportPopup()` below, did not work. i think the imgui functions have to be called during `Draw()`? not completely sure. but this does work.